Week1/Homework/js-exercises/compareArrays.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+var me = ["Abuzer", 29, "unemployed", "bachelor graduate"];
+
+var tesla = [
```

Comment:
Nice, I think a more descriptive name could be "teslaModels" (especially because of the **s** because an array contains multiple items, so usually the variable is in the plural:))

Reply:
Ok I will change the name thanks a lot